demo/demo_director.py

```diff
@@ -190,16 +193,21 @@ def write_to_live(vin_to_update=None):
         os.path.join(repo_dir, 'metadata.livetemp'),
         os.path.join(repo_dir, 'metadata'))
 
+    if backup_repo:
+      # Copy the staged metadata to a backup directory, which we'll move back
+      # into place when we later undo sign_with_compromised_keys_attack().
+      shutil.copytree(os.path.join(repo_dir, 'metadata.staged'),
+          os.path.join(repo_dir, 'metadata.backup'))
```

Comment:
Sure.  I will work to reproduce the unexpected ReplayedMetadataError and determine the cause and fix.